datalad/metadata/__init__.py

```diff
@@ -321,10 +321,17 @@ def _cached_load_document(url):
             urlsplit(url).netloc,
             md5(url.encode('utf-8')).hexdigest()))
 
+    doc = None
     if os.path.exists(doc_fname):
-        lgr.debug("use cached request result to '{}' from {}".format(url, doc_fname))
-        doc = pickle.load(open(doc_fname))
-    else:
+        try:
+            lgr.debug("use cached request result to '%s' from %s", url, doc_fname)
+            doc = pickle.load(open(doc_fname))
+        except Exception as e:  # it is OK to ignore any error and fall back on the true source
+            lgr.warning(
+                "cannot load cache from '%s', fall back on schema download: %s",
+                doc_fname, exc_str(e))
+            pass
```

Comment:
pass is not needed but that is ok ;-)  someone will need to make a test eventually as well anyways so that someone will clean up (the worst exceptions are the ones thrown in exception handlers ;)